src/test/java/com/xebialabs/xlt/ci/server/XLTestServerImplTest.java

```diff
@@ -1,65 +1,93 @@
 package com.xebialabs.xlt.ci.server;
```

Comment:
The reason was that I tried to get the @JenkinsRule to work, which is a JUnit rule that boots a jenkins instance and allows integration testing. It would be very useful, but it was difficult, and I ended up abandoning it. This was a relic that stayed in here. I agree it should have been done in a different PR